code/modules/games/cah.dm

```diff
@@ -0,0 +1,262 @@
+	icon_state = "cag_black"
+	blanks = 0
+
+/obj/item/deck/cah/New()
+	..()
+	var/datum/playingcard/P
+	for(var/cardtext in card_text_list)
+		P = new()
+		P.name = "[cardtext]"
+		P.card_icon = "[icon_state]_card"
+		P.back_icon = "[icon_state]_card_back"
+		cards += P
+	if(!blanks)
+		return
+	for(var/x=1 to blanks)
+		P = new()
+		P.name = "Blank Card"
+		P.card_icon = "[icon_state]_card_back"
+		P.back_icon = "[icon_state]_card_back"
+
+// Black cards.
+/obj/item/deck/cah/black/card_text_list = list(
+	"Why am I itchy?",
+	"Today, police shot ____.",
+	"Our local chaplain is worshipping _____.",
+	"My grandparents ordered a crate full of _____ from the catalogue.",
+	"The FIB was called due to ______.",
+	"Alert! The Sol Navy has armed themselves with _____.",
+	"Order received: ________ is your master.",
+	"Order received: ________ is the enemy.",
+	"_____ blew a hole in the wall.",
+	"Today, researchers found an artefact that made people ____ and ____. (Pick 2)",
+	"There was a rap battle between ____ and ____. (Pick 2)",
+	"Caution, ______ have been detected in collision course with the planet.",
+	"Today's kitchen menu includes _______.",
+	"What did space pirates want when they attacked our home?",
+	"I think our boss is insane. He just demanded ______ in his office.",
+	"Fuckin' scientists, they just turned the entire population of Scottsdan into _______ .",
+	"Fuckin' activists, they just converted Mendell City into ______ .",
+	"What's my fetish?",
+	"Next up on TCFL's Got Talent: ________ .",
+	"Next up on Biesel's World Cooking Championships: _______ .",
+	"Hello, _______ here with _______ . (Pick 2)",
+	"No one else was at _______, they wouldn't understand the ______ . (Pick 2)",
+	"Why am I shivering?",
+	"What is this world coming to? First, ________, now _______ . (Pick 2)",
+	"NanoTrasen's morale division decided to use _______ to raise employee morale.",
+	"The average junkie's drug of choice is ______ .",
+	"I saw an Eridanian dreg make out with ______ the other day.",
+	"It is common practice for _______ to ______ on Moghes. (Pick 2)",
+	"The colonies of the Coalition of Colonies are really just _____.",
+	"The Skrell are celebrating _____ today.",
+	"_____ is/are why I'm afraid of hospitals.",
+	"_____ used ____ to create their newest invention, _____!  (Draw 2; Pick 3)",
+	"Scientists are not allowed to make Gatling _____.",
+	"It's not a party until the ____ arrive.",
+	"No matter how many Tajara you have, _____ is never acceptable.",
+	"No, your robot's first law is NOT to serve _____.",
+	"The robots are not disposal bins for your _____.",
+	"You can never have too many _____ on board.",
+	"Failing to fit ______ up the _____ . (Pick 2)",
+	"_____ has no idea what S&M is and _____ shows them. (Pick 2)",
+	"What's this in my hand?",
+	"My lover left me for ______.",
+	"The _____ apocalypse comes.",
+	"Hi! I'm Malyr Krissen, and I got hit by ______.",
```

Comment:
Who is Malyr Krissen.

Reply:
Made-up. I was hoping it'd be a prompt for players to determine what that was about—you get plenty of people who do this and I can provide the logs if you would allow me a moment to retrieve them if you wish for evidence to back up this claim.